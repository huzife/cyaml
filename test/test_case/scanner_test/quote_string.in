---
quote_str: 'this    
 is a
                    test string'
next: null
