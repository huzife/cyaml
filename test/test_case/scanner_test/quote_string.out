(DOC_START)#
(BLOCK_MAP_START)#
(KEY)#
(SCALAR, quote_str)#
(VALUE)#
(SCALAR, this     is a test string)#
(KEY)#
(SCALAR, next)#
(VALUE)#
(KEY)#
(SCALAR, last)#
(VALUE)#
(BLOCK_MAP_END)#
